### PR TITLE
DR-2353 add run reason and skip reason

### PR DESCRIPTION
**Pull request by tcjiang9**

### CODE REVIEW

File: src/main/java/bio/terra/service/dataset/flight/ingest/CombinedFileIngestOptionalStep.java [hunk 1]
```diff
@@ -15,7 +16,16 @@ public boolean isEnabled(FlightContext context) {
...
     return "there are no files to ingest";
   }
+
+  @Override
+  public String getRunReason(FlightContext context) {
+    long numFiles =
+        Objects.requireNonNullElse(
+            context.getWorkingMap().get(IngestMapKeys.NUM_BULK_LOAD_FILE_MODELS, Long.class),
+            0L);
+    return numFiles + "bulk load files found to ingest";
```
Comment (nmalfroy) [inline src/main/java/bio/terra/service/dataset/flight/ingest/CombinedFileIngestOptionalStep.java]: you need an extra space here.  I'd recommend using string format. e.g.
```suggestion
    return String.format("%s bulk load file(s) found to ingest", numFiles);
```

**Review by nmalfroy — CHANGES_REQUESTED** [review]
Small things and I think the linting needs a pass

**Comment by nmalfroy** [comment]
Also, can you add the jira ticket to the PR?

**Review by snf2ye — APPROVED** [review]
Given @nmalfroy's comments are addressed, LGTM

**Review by nmalfroy — APPROVED** [review]

**Pull request merged**